virttest/qemu_devices/qcontainer.py

```diff
@@ -2255,7 +2255,11 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         if 'socket' in backend:  # tcp_socket & unix_socket
             chardev_param.update(
                 {'server': params.get('chardev_server', 'yes'),
-                 'nowait': params.get('chardev_nowait', 'yes')})
+                 'nowait': params.get('chardev_nowait', 'yes'),
+                 'host': params.get('chardev_host', ''),
+                 'port': params.get('chardev_port', ''),
+                 'to': params.get('chardev_to','')
```

Comment:
Only tcp socket have option `to`

Reply:
And, `chardev_host` and `chardev_port` value are setted in qemu_vm.py, and used in L2251~L2252

Reply:
hi in virttest/qemu_vm.py L2251~L2252, it's                 
devs[-1].set_param("if", "pflash")
devs[-1].set_param("format", "raw")
This is the only qemu_vm.py file I can found

Reply:
@michiboo     https://github.com/avocado-framework/avocado-vt/blob/e2d51980f53abc63810c2813b256a4c1e4adadb8/virttest/qemu_vm.py#L1771